apps/front-client/src/features/sidebar/components/Molecules/Profile.tsx

```diff
@@ -25,6 +27,21 @@ const ProfileDescription = styled(Box)`
 `;
 
 const Profile = () => {
+  const session = useSession();
+  const user = session?.data?.user;
+
+  if (!user) {
+    return <p>User not found or not logged in</p>;
+  }
```

Comment:
Okay alors ici en vrai je retournerais juste `null`, parce qu'au final le seul moment où tu rentres dans ce cas c'est quand la data charge, et vaut mieux ne rien afficher qu'un message un peu faux

```suggestion
  if (!user) {
    return null;
  }
```